### ISSUE TITLE
Bug: Crash in attempting a redirect

### ISSUE THREAD

**Issue by Not-Alaksa-Airlines** [face-hh/webx#94]
I had opened the registrar but wanted to return to the search engine main page. I didn't remember the URL, so I simply deleted the URL from the search bar. It said that it couldn't find the website (of course, none was requested) but when I clicked on the link to take me to the browser, the whole browser froze and then crashed after a few seconds.

**Comment by Not-Alaksa-Airlines** [comment]
Now, I retried, and it didn't repeat, so I don't know what's happening, but if I can help find the issue, I would be happy to.

**Comment by alexandre-lavoie** [comment]
I also noticed that navigating between pages can be pretty taxing on my system at times. I haven't profiled the app but I *think* that probably having a lot of objects be deleted and created at once may be causing this. You potentially were about to run out of memory and your OS decided to save you (if you had the crash logs it would interesting to see!)

**Comment by Not-Alaksa-Airlines** [comment]
> I also noticed that navigating between pages can be pretty taxing on my system at times. I haven't profiled the app but I _think_ that probably having a lot of objects be deleted and created at once may be causing this. You potentially were about to run out of memory and your OS decided to save you (if you had the crash logs it would interesting to see!)

Do you know where I might be able to find crash logs? I'm not sure if any were recorded because it crashed without even bringing up the prompt for waiting for the program to respond or terminating the process.

**Comment by GStudiosX2** [comment]
> I also noticed that navigating between pages can be pretty taxing on my system at times. I haven't profiled the app but I *think* that probably having a lot of objects be deleted and created at once may be causing this. You potentially were about to run out of memory and your OS decided to save you (if you had the crash logs it would interesting to see!)

I believe the reason for high usage is because it blocks

**Comment by GStudiosX2** [comment]
> > I also noticed that navigating between pages can be pretty taxing on my system at times. I haven't profiled the app but I _think_ that probably having a lot of objects be deleted and created at once may be causing this. You potentially were about to run out of memory and your OS decided to save you (if you had the crash logs it would interesting to see!)
> 
> Do you know where I might be able to find crash logs? I'm not sure if any were recorded because it crashed without even bringing up the prompt for waiting for the program to respond or terminating the process.

I don't think you can do it on windows maybe if you build debug? But if you open it in a terminal you should get any logs and panic errors
